airflow/providers/tabular/CHANGELOG.rst

```diff
@@ -26,6 +26,13 @@
 Changelog
 ---------
 
+1.5.0
+.....
```

Comment:
```suggestion
```

No need for the version. Release manager auto pick up the comment at the change log and place it to the chosen release version